README.md

```diff
@@ -62,47 +61,42 @@ To know more about the architectural details, please read [here](design/design_d
 
 ## 1. Quick Start
 
-There are multiple installation paths for installing FfDL locally ("1-click-install") or into an existing Kubernetes cluster.
+There are multiple installation paths for installing FfDL locally ("1-click-install") or into an existing Kubernetes cluster. You can visit [Step 5](#5-detailed-installation-instructions) for more details on the deployment instructions.
 
-> Note: If your Kubernetes Cluster version is 1.7 or below, please go to the [values.yaml](values.yaml) and change `k8s_1dot8_or_above` to **false**.
+### 1.1 Installation using Kubeadm-DIND
 
-### 1.1 Installation using Minikube
-
-If you have Minikube installed on your machine, use these commands to deploy the FfDL platform:
+If you have [Kubeadm-DIND](https://github.com/kubernetes-sigs/kubeadm-dind-cluster#using-preconfigured-scripts) installed on your machine, use these commands to deploy the FfDL platform:
 ``` shell
-export VM_TYPE=minikube
-make minikube
-make deploy
+export VM_TYPE=dind
+export PUBLIC_IP=localhost
+export SHARED_VOLUME_STORAGE_CLASS="";
+make deploy-plugin
+make quickstart-deploy
 ```
 
 ### 1.2 Installation using Kubernetes Cluster
 
-To install FfDL to a proper Kubernetes cluster, make sure `kubectl` points to the right namespace,
+To install FfDL to any proper Kubernetes cluster, make sure `kubectl` points to the right namespace,
 then deploy the platform services:
-> Note: For PUBLIC_IP, put down one of your Cluster Public IP that can access your Cluster's NodePorts.
+> Note: For PUBLIC_IP, put down one of your Cluster Public IP that can access your Cluster's NodePorts. For IBM Cloud, you can get your Public IP with `bx cs workers <cluster_name>`.
 
 ``` shell
 export VM_TYPE=none
 export PUBLIC_IP=<Cluster Public IP>
-make deploy
-```
-
-### 1.3 Installation using IBM Cloud Kubernetes Cluster
 
-To install FfDL to a proper IBM Cloud Kubernetes cluster, make sure `kubectl` points to the right namespace
-and your machine is logged in with `bx login`, then deploy the platform services:
-``` shell
-export VM_TYPE=ibmcloud
-export CLUSTER_NAME=<Your Cluster Name> # Replace <Your Cluster Name> with your IBM Cloud Cluster Name
-make deploy
+# Change the storage class to what's available on your Cloud Kubernetes Cluster.
+export SHARED_VOLUME_STORAGE_CLASS="ibmc-file-gold";
+make deploy-plugin
```

Comment:
are we missing a `helm init` before `make deploy-plugin` here? without that I get `Error: could not find a ready tiller pod`:
```
[ffdl]$ make deploy-plugin

Deploying services to Kubernetes. This may take a while.
Installing helm/tiller
Error: could not find a ready tiller pod
Wait while kubectl get pvc shows static-volume-1 in state Pending
Creating persistent volume claim 1
Error: could not find a ready tiller pod
persistentvolumeclaim "static-volume-1" created
Error from server (NotFound): configmaps "static-volumes" not found

Using volumes with label type=dlaas-static-volume:
NAME              STATUS    VOLUME    CAPACITY   ACCESS MODES   STORAGECLASS     AGE
static-volume-1   Pending                                       ibmc-file-gold   1s

configmap "static-volumes" created

```

Reply:
Oh, sorry we did add helm init, but we forgot to put a sleep loop to wait for tiller to be ready.